docker-compose.yml

```diff
@@ -1,14 +1,19 @@
 ---
 version: "3.7"
+x-orchestrator-config:
+  POSTGRES_USER: ${POSTGRES_USER:-postgres_user}
+  POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:-postgres_pass}
+  POSTGRES_DB: ${POSTGRES_DB:-postgres_db}
+  INFRAHUB_TOKEN: ${INFRAHUB_TOKEN:-"06438eb2-8019-4776-878c-0941b1f1d1ec"}
 
 services:
   dagster_postgresql:
     image: postgres:11
     container_name: dagster_postgresql
     environment:
-      POSTGRES_USER: "postgres_user"
-      POSTGRES_PASSWORD: "postgres_password"
-      POSTGRES_DB: "postgres_db"
+      POSTGRES_USER: ${POSTGRES_USER}
```

Comment:
If you split `x-orchestrator-config`, you can reuse it without having to redefine the variables here

Reply:
Should be good now